server/src/test/java/org/elasticsearch/action/search/SearchQueryThenFetchAsyncActionTests.java

```diff
@@ -258,11 +258,10 @@ public void run() {
         }
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/101932")
     public void testMinimumVersionSameAsNewVersion() throws Exception {
         var newVersion = VersionInformation.CURRENT;
         var oldVersion = new VersionInformation(
-            VersionUtils.randomVersionBetween(random(), Version.CURRENT.minimumCompatibilityVersion(), VersionUtils.getPreviousVersion()),
+            VersionUtils.randomCompatibleVersion(random(), VersionUtils.getPreviousVersion()),
```

Comment:
This test hasn't been running on any major version bump as far as I can tell (it has been muted a long time). On the v8 series, Version.CURRENT.minimumCompatibilityVersion() was always 7.17 and the previous-to-Current has always been some 8.x minor version.
This doesn't work with the major version bump at the moment since Current is 9.0, previous-to-Current is 8.15.2 (!!!) due to 8.16 not having been released yet, and minimum compatible is 8.16. So currently this errors.
I believe what we want in this test is "newVersion" at whatever the main branch is currently on (9.x minors), and oldVersion anything from minimumCompatible up to previous. For now this would only be 8.16, but as soon as we move on to other 9x minor versions we draw from a larger range.
So I think we are good here, but please keep me honest.

Reply:
Yes, your explanation aligns with my understanding, thank you for double-checking this!